src/azure-cli/azure/cli/command_modules/apim/_help.py

```diff
@@ -373,6 +378,16 @@
         az apim nv update --service-name MyApim -g MyResourceGroup --named-value-id MyNamedValue --value foo
 """
 
+helps['apim nv wait'] = """
+type: command
+short-summary: Place the CLI in a waiting state until a condition of an apim named value is met.
+examples:
+  - name: Place the CLI in a waiting state until a condition of a apim api is met. (autogenerated)
+    text: |
+        az apim nv wait --created --service-name MyApim -g MyResourceGroup --named-value-id MyNamedValue --resource-group MyResourceGroup
+    crafted: true
+"""
```

Comment:
If this is manually added help message, please do not add these labels: `(autogenerated)` and `crafted: true`
```suggestion
helps['apim nv wait'] = """
type: command
short-summary: Place the CLI in a waiting state until a condition of an apim named value is met.
examples:
  - name: Place the CLI in a waiting state until a condition of a apim api is met.
    text: |
        az apim nv wait --created --service-name MyApim -g MyResourceGroup --named-value-id MyNamedValue --resource-group MyResourceGroup
"""
```